pypeit/spectrographs/gemini_gmos.py

```diff
@@ -333,7 +333,7 @@ def get_rawimage(self, raw_file, det):
             array[row1-nodpix:row2-nodpix,:] = array[row1:row2,:]
 
         # Return, transposing array back to orient the overscan properly
-        return self.get_detector_par(hdu, det if det is None else 1), \
+        return self.get_detector_par(hdu, det if det is not None else 1), \
```

Comment:
good catch.  I think it started in `keck_lris` which I have
fixed previously but hadn't realized it had propogated..

Reply:
like a virus..